src/libs/actions/User.js

```diff
@@ -568,6 +569,16 @@ function subscribeToUserEvents() {
             if (!currentUserAccountID) {
                 return;
             }
+
+            // If there is an OnyxUpdate for using memory only keys, enable them
+            _.each(pushJSON, ({key, value}) => {
+                if (key !== ONYXKEYS.IS_USING_MEMORY_ONLY_KEYS || !value) {
+                    return;
+                }
+
+                MemoryOnlyKeys.enable();
+            });
```

Comment:
NAB, I thought maybe we were just sending these `onyxUpdate` with this key via https? Is including the check here for Pusher updates sort of a future proofing?

Reply:
Yeah, you're right. This is a hold-over from my original testing/implementation but it doesn't need to be here and is a little confusing. I'll remove it.